dessia_common/core.py

```diff
@@ -638,10 +638,28 @@ def to_xlsx_stream(self, stream):
         writer.save_to_stream(stream)
 
     def _export_formats(self):
-        formats = [{"extension": "json", "method_name": "save_to_stream", "text": True, "args": {}},
-                   {"extension": "xlsx", "method_name": "to_xlsx_stream", "text": False, "args": {}}]
+        formats = [{"selector": "json", "extension": "json",
```

Comment:
Workflows now define an export_format with an "export_name" entry (that could be renamed "filename"), in order to set a by default filename known by frontend for an improved experience while exporting stuff.

Should we set it in this generic definition ? If yes, which names could we compute by default ?